recipes/pre-commit/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "pre-commit" %}
+{% set version = "1.12.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "81fe7b1b0cba93e7d7aa3cfbfd55f5280d3639e4fcb14bf9d1b95ab35f41cf0d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-v{{ version }}.tar.gz
+  url: https://github.com/pre-commit/pre-commit/archive/v{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - pre-commit = pre_commit.main:main
+    - pre-commit-validate-config = pre_commit.clientlib:validate_config_main
+    - pre-commit-validate-manifest = pre_commit.clientlib:validate_manifest_main
+
+requirements:
+  build:
```

Could you please change it to host?
```suggestion
  host:
```

---

Done!